Explorer/Assets/DCL/ApplicationMinimumSpecsGuard/UI/MinimumSpecsTablePresenter.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using DCL.Diagnostics;
+
+namespace DCL.ApplicationMinimumSpecsGuard
+{
+    public class MinimumSpecsTablePresenter
+    {
+        private const string PASS_ICON_SPRITE_TAG = "<sprite name=\"2705\">";
+        private const string FAIL_ICON_SPRITE_TAG = "<sprite name=\"274c\">";
+        
+        private readonly Dictionary<SpecCategory, MinimumSpecsRowView> rowMap;
+
+        public MinimumSpecsTablePresenter(MinimumSpecsTableView view)
+        {
+            rowMap = new Dictionary<SpecCategory, MinimumSpecsRowView>();
+
+            foreach (var row in view.Rows)
```

Comment:
You are using the rows to determine how many categories you are going to display. A row view shouldn't even know what category they belong to, they should just display strings. 

I do think this should be the other way around, the categories should come from the controller. You could find the way to hook it from `MinimumSpecsGuard`

Whatsmore, I do think that the rows should be dynamically instantiated, the prefab should have no rows at all.